tools/git_hooks/find_duplicates.py

```diff
@@ -1,26 +0,0 @@
-from os import walk
```

Comment:
Where was this file being used?

Reply:
It was not, unless you added it as a git hook.